src/download_games/get_games.py

```diff
@@ -149,6 +169,7 @@ def download_games(name: str,
 
     # this is a mock to demo how to use this function, the timeframes are very small
     download_games(user_name,
+                   db_dir="../../resources",
```

Comment:
These hardcoded paths look dangerous.

Reply:
replaced with reference to project's root path.